samples/sampler/stories/default/About.js

```diff
@@ -85,7 +85,7 @@ export const ATourOfSampler = (args) => (
 		>
 			Select any component from the <b>sidebar</b> to see how it works
 		</HintDialog>
-		<HintDialog style={{bottom: riSafe(edgeDotKeepout), left: 39, top: 480}} length={540}>
+		<HintDialog style={{bottom: riSafe(edgeDotKeepout), left: 39}} length={348}>
```

Comment:
How does come the 348 value? 
Why only bottom wrapped by riSafe? Why left is not wrapped?

Reply:
348 is a heuristic value that makes hint dialogs do not overlap each other.
I refered prior length of line 98. The hint dialog of line 98 didn't overlap with the one of line 81.
[bottom] should be wapped by riSafe because the size of hint dialog changes when you expand the side bar left to right.
But [left] position should not because the position of [Actions] and [Controls] buttons maintains their size.

Reply:
As I tested, hint dialogs still overlap each other. Could you please check if it can be fixed?